Support for new clauses and bug/conflicts fixes

carlasanches opened the pull request:
This PR solves conflicts with PostgreSQL clauses, removes warning messages, adds support for new clauses, and fixes segmentation fault errors.

ibrarahmad reviewed (changes requested):
Please remove conflicts.

carlasanches committed:
Support for new clauses and bug/conflicts fixes

This commit solves conflicts with PostgreSQL clauses, removes warning messages, adds support for new clauses, and fixes segmentation fault errors.
